### PR TITLE
[Fix] Test graph construction and fix a few naming bugs

### PR DESCRIPTION
**Pull request by zhjwy9343**
*Issue #, if available:*
The construct_graph.py has some naming bugs. 

*Description of changes:*

- Add a new dummy customer data generation codes

- Fix a few naming issues in construct_grapy.py and data_gen.py.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/end2end-tests/data_process/gen_test_graphs.py [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+    e_dfs_dict[('n0', 'e0', 'n1')] = e1_df
+
+    # -------- build edge2 between node1 and node3 ---------
+    e2_src_ids = np.random.choice(n1_ids, NUM_ETYPE2 * factor + 100, replace=True)
+    e2_dst_ids = np.random.choice(n3_ids, NUM_ETYPE2 * factor + 100, replace=True)
+    e2_df = pd.DataFrame({'src_id': e2_src_ids,
+                          'dst_id': e2_dst_ids})
+    e2_df = e2_df.drop_duplicates().iloc[ :NUM_ETYPE2 * factor]
+    
+    # [TODO] James: Keep these commented codes for later test after construct graph fix this issue.
```
Comment (zheng-da) [inline tests/end2end-tests/data_process/gen_test_graphs.py:119]: what is the issue?

File: tests/end2end-tests/data_process/gen_test_graphs.py [hunk 2]
```diff
@@ -0,0 +1,374 @@
+"""
```
Comment (zheng-da) [inline tests/end2end-tests/data_process/gen_test_graphs.py:1]: this script is not used in CI. I wonder if we need to have such a file checked in the test folder.

Comment (zheng-da) [reply tests/end2end-tests/data_process/gen_test_graphs.py:1]: we already have a script that generates a test graph. https://github.com/awslabs/graphstorm/blob/main/tests/end2end-tests/data_process/data_gen.py
we run the pipeline on the test graph and verify the correctness of the partition result.

Comment (zhjwy9343) [reply tests/end2end-tests/data_process/gen_test_graphs.py:1]: This script will be used for regression test.

**Review by zheng-da — APPROVED** [review]

File: tests/end2end-tests/data_process/gen_test_graphs.py (hunk 1 above)
Comment (zhjwy9343) [reply tests/end2end-tests/data_process/gen_test_graphs.py:119]: This is the issue about only partial nodes have labels.

**Pull request merged**